extract/kbs_crawler.py

```
@@ -0,0 +1,90 @@
+from typing import list, TypedDict
```

Ruff (0.8.2)

`typing.TypedDict` imported but unused

Remove unused import: `typing.TypedDict`

(F401)